### DISCUSSION TITLE
Latest cjose file only in v 2.4.0 ?

### DISCUSSION THREAD

**Discussion by rpa275** [OpenIDC/mod_auth_openidc#844] [Q&A] [answered]
The latest cjose file is only available in old version 2.4.0. Noticed that this file under the assets is getting updated to different versions, causing the previous version to be erased. Is there a better place to retrieve the different versions of cjose or get the latest cjose which is updating under the v2.4.0 right now?

Use to be cjose-0.6.1.5-1.el7.x86_64.rpm and now it changed to cjose-0.6.1.5-2.el7.x86_64.rpm

This is for Centos7

Thanks!

**Comment by zandbelt** [comment, marked as answer]
yeah, there was a packaging update that fixes a long-standing dependency problem; but by now most distributions carry their own, pretty recent version of cjose so the packages in 2.4.0 are not so much needed anymore; on top of that, there's now https://github.com/zmartzone/cjose/releases/tag/v0.6.2 which should be in the repositories sooner or later as well (see also https://github.com/cisco/cjose/issues/121)
